atest/acceptance/keywords/click_element.robot

```diff
@@ -36,6 +36,12 @@ Double Click Element Error
     [Setup]    Go To Page "javascript/click.html"
     Double Click Element    id:öööö
 
+Click Element Action Chain
+    [Documentation]   Clicks an element using the 'action_chain' argument without a modifier
+    ...               LOG 2     REGEXP: Clicking '.*' using an action chain.
```

Comment:
Better but not quite on what I had in my mind. Test are done, among other things, to safe guard maintainers from stupid mistakes. Now I can change the action chains implementation and it's not always certain that test would capture the accidental change. 

The test now ensures that keyword logs by using the Robot Framework API (which is relevant too) and some sort of click happens, but it doesn't ensure that action chains click happens. 

Please add more checking that keyword calls the correct Selenium API methods.

Same comment applies also to the test in below.

Reply:
True. It might come down to a new unit test. Unfortunately, ActionChains click() and WebElement.click() register the same JavaScript MouseEvent action so I can't tell them apart that way. I'll see what I can do.

Reply:
I'm really at a loss for what I can do with this. I've tried a whole bunch of different things with mockito's spy, spy2, patch, etc. and I can't seem to figure out something that'll work. Do you have any guidance here?